pkg/console/console.go

```diff
@@ -58,11 +65,41 @@ func Run() error {
 	restful.Filter(cors.Filter)
 	restful.Filter(restful.OPTIONSFilter())
 
+	tlsConfigWatch := tlsconfig.NewWatch(
+		path.Join(configDir, TlsProfileFile),
```

Comment:
Would using `filepath` instead of `path` be an improvement?

Reply:
done